app/Models/Items.php

```diff
@@ -6,12 +6,17 @@
 
 class Items
```

Comment:
OOP Object is one entity not multiple, each Item is different object, and as whole they are collection of objects

Reply:
Same as you have a car, and in garage there are multiple cars